source/_posts/2019-09-27-new-blog-psr-12-approval-and-upgrading-psr.md

```diff
@@ -0,0 +1,82 @@
+---
+title: 'New blog, PSR-12 approval and Upgrading PSRs'
+description: >-
+ And we’re back with another update on what’s going on in the PHP-FIG! This
+ time we have just two news, but big ones! Since the last update, we’ve seen
+ the approval of PSR-14, the Event Dispatcher…
+date: '2019-09-27T14:11:13.604Z'
+categories: []
+keywords: []
+slug: updates-from-the-php-fig-up-until-the-may-elections
+tags:
+ - php-fig
+ - psr-14
+ - event-dispatcher
+author: alessandrolai
+layout: post
+use:
+ - authors
+ - posts
+---
+
+After a long iatus, we're back with a new round of updates from the PHP-FIG.
+
+## New Blog
+
+As you may have noticed, **we're now on a different platform**. We've chosen to migrate away from the previous proprietary platform and to **host directly our posts inside our site**, directly under our control. We will add links at the top of all our older posts to the new copies here, so if anyone still stumbles upon our posts on the previous platform, they will know we're to find us from now on. 
```

Comment:
I would split this phrase in two, on the comma before "so if anyone". It took me some time ingest the phrase